src/phasorpy/phasor.py

```diff
@@ -435,102 +464,139 @@ def phasor_calibrate(real, imag, reference_real, reference_imag, frequency, life
         The array shapes of `real` and `imag`, or `reference_real` and
         `reference_imag` do not match.
 
+    Notes
+    -----
+    This function is a convenience wrapper for the following operations:
+
+    phasor_transform(
+        real,
+        imag,
+        *polar_from_reference_phasor(
+            *phasor_center(
+                reference_real,
+                reference_imag,
+                skip_axes,
+                method,
+            ),
+            *phasor_from_lifetime(
+                frequency,
+                lifetime,
+                fraction,
+                preexponential,
+                unit_conversion,
+            ),
+        ),
+    )
```

Comment:
Use a block:
```
    This function is a convenience wrapper for the following operations::

        phasor_transform(
            real,
            imag,
            *polar_from_reference_phasor(
                *phasor_center(
                    reference_real,
                    reference_imag,
                    skip_axes,
                    method,
                ),
                *phasor_from_lifetime(
                    frequency,
                    lifetime,
                    fraction,
                    preexponential,
                    unit_conversion,
                ),
            ),
        )
```

Reply:
Or better, use a [Sphinx code_block](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/code_blocks.html).

Reply:
Or better use a [Sphinx code-block](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block)